05-cond.md

```diff
@@ -86,7 +86,7 @@ We can also combine tests using `and` and `or`.
 `and` is only true if both parts are true:
 
 ~~~ {.python}
-if (1 > 0) and (-1 > 0):
+if 1 > 0 and -1 > 0:
```

Comment:
I've seen people get confused when there are no parentheses: `1 > 0 and -1` can be parsed as `1 > (0 and -1)` -- unless you've already read the whole line. I tend to use parentheses because they improve *reading* code, which I think is worth emphasizing.

If we want to not use the parentheses, maybe there needs to be something on the precedence of operators, and I think this lesson is dense already.

I think the parentheses should stay.